agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/templates/notification/push/VotePushPlugin.gtmpl

```diff
@@ -0,0 +1,14 @@
+<% eventTitle = _ctx.escapeHTML(eventTitle); %>
+<div>
+    <div class="contentSmall" data-link="<%= eventURL %>">
+        <div class="status">
+            <%
+            String eventTitle = "<span style=\"color: #2f5e92; text-decoration: none;font-weight:bold; font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif\" >" + eventTitle + "</span>";
+            String agendaName = "<a style=\"text-decoration: none; font-weight:bold;font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif\" >" + agendaName +"</a>";
+            String participantName = "<a style=\" text-decoration: none; font-weight:bold;font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif\" >" + participantName +"</a>";
+            %>
+            <%= _ctx.appRes("Notification.agenda.event.mail.body.date.vote", participantName, eventTitle, agendaName) %>
```

Comment:
The I18N key isn't good, you used `Notification.agenda.event.mail.body.date.vote`